src/templates/organization.js

```diff
@@ -153,7 +155,10 @@ export default function OrganizationTemplate({ data }) {
         <div className="mb-10 md:mt-6 flex flex-col lg:flex-row">
           <div className="w-5/5 lg:w-3/5">
             <div className="flex">
-              <div className="mr-5 w-16 flex-shrink-0 block">
+              <div
+                className="mr-5 w-16 flex-shrink-0 block"
+                style={{ display: showImg }}
+              >
                 {img && (
```

Comment:
It looks like this logic is already here, just in the wrong place. If we can just move this up to the `<div>` instead of the `<Img>` tag it will fix the issue without needing to hide anything with CSS.

Reply:
ah yes totally, nice catch